dbfit-java/sqlserver/src/main/java/dbfit/environment/SqlServerEnvironment.java

```diff
@@ -208,6 +216,13 @@ private static int getSqlType(String dataType) {
 
     public Map<String, DbParameterAccessor> getAllProcedureParameters(
             String procName) throws SQLException {
+
+        String objectDatabaseName = "";
+        String[] objnameParts = procName.split("\\.");
+        if (objnameParts.length == 3) {
+            objectDatabaseName = objnameParts[0] + ".";
+        }
```

Comment:
That snippet is basically the same as the other. Maybe we can move it to a single place?